script/validators_update_avatar_url.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# RAILS_ENV=production bundle exec ruby script/validators_get_avatar_url.rb
+require File.expand_path('../config/environment', __dir__)
+require File.expand_path('./app/services/keybase_service.rb')
+# Default URL is at 'https://keybase.io/images/no-photo/placeholder-avatar-180-x-180@2x.png'
+
+# include AgentLogic
+# include KeyBaseService
+
+interrupted = false
+trap('INT') { interrupted = true }
+
+agent = Mechanize.new
```

Comment:
Might be overkill to use Mechanize here. `Net::HTTP` or `RestClient` might be better.